android/app/build.gradle

```diff
@@ -232,63 +231,32 @@ dependencies {
-  api 'host.exp.exponent:expo-constants:4.0.0'
-  api 'host.exp.exponent:expo-document-picker:4.0.0'
-  api 'org.unimodules:unimodules-file-system-interface:1.0.0'
-  api 'host.exp.exponent:expo-file-system:4.0.0'
-  api 'org.unimodules:unimodules-image-loader-interface:1.0.0'
-  api 'host.exp.exponent:expo-permissions:4.0.0'
-  api 'org.unimodules:unimodules-permissions-interface:1.0.0'
-  api 'org.unimodules:unimodules-sensors-interface:1.0.0'
-  api 'org.unimodules:unimodules-react-native-adapter:1.0.0'
-  api 'host.exp.exponent:expo-task-manager:4.0.0'
-  api 'org.unimodules:unimodules-task-manager-interface:1.0.0'
-
-  // Optional universal modules, could be removed
-  // along with references in MainActivity
-  api 'host.exp.exponent:expo-ads-admob:4.0.0'
-  api 'host.exp.exponent:expo-ads-facebook:4.0.0'
-  api 'host.exp.exponent:expo-analytics-amplitude:4.0.0'
-  api 'host.exp.exponent:expo-app-auth:4.0.0'
-  api 'host.exp.exponent:expo-av:4.1.0'
-  api 'host.exp.exponent:expo-analytics-segment:4.0.0'
-  api 'org.unimodules:unimodules-barcode-scanner-interface:1.0.0'
-  api 'host.exp.exponent:expo-barcode-scanner:4.0.0'
-  api 'host.exp.exponent:expo-brightness:4.0.0'
-  api 'host.exp.exponent:expo-calendar:4.0.0'
-  api 'org.unimodules:unimodules-camera-interface:1.0.0'
-  api 'host.exp.exponent:expo-camera:4.0.0'
-  api 'host.exp.exponent:expo-contacts:4.0.0'
-  api 'host.exp.exponent:expo-facebook:4.0.0'
-  api 'host.exp.exponent:expo-face-detector:4.0.0'
-  api 'org.unimodules:unimodules-face-detector-interface:1.0.0'
-  api 'host.exp.exponent:expo-font:4.0.0'
-  api 'host.exp.exponent:expo-gl-cpp:4.0.0'
-  api 'host.exp.exponent:expo-gl:4.0.0'
-  api 'host.exp.exponent:expo-google-sign-in:4.0.0'
-  api 'host.exp.exponent:expo-haptics:4.0.0'
-  api 'host.exp.exponent:expo-image-manipulator:4.0.0'
-  api 'host.exp.exponent:expo-image-picker:4.0.0'
-  api 'host.exp.exponent:expo-intent-launcher:4.0.0'
-  api 'host.exp.exponent:expo-local-authentication:4.0.0'
-  api 'host.exp.exponent:expo-localization:4.0.0'
-  api 'host.exp.exponent:expo-location:4.0.0'
-  api 'host.exp.exponent:expo-media-library:4.0.0'
-  api 'host.exp.exponent:expo-print:4.0.0'
-  api 'host.exp.exponent:expo-secure-store:4.0.0'
-  api 'host.exp.exponent:expo-sensors:4.0.0'
-  api 'host.exp.exponent:expo-sharing:2.0.0'
-  api 'host.exp.exponent:expo-sms:4.0.0'
-  api 'host.exp.exponent:expo-linear-gradient:4.0.0'
-  api 'host.exp.exponent:expo-keep-awake:4.0.0'
-  api 'host.exp.exponent:expo-background-fetch:4.0.0'
-  api 'host.exp.exponent:expo-mail-composer:4.0.0'
-  api 'host.exp.exponent:expo-speech:4.0.0'
-  api 'host.exp.exponent:expo-sqlite:4.0.0'
-  api 'host.exp.exponent:expo-web-browser:4.0.0'
+
+  addMavenUnimodulesDependencies([
+      modulesPaths : [
+        '../enabled-modules'
+      ],
+      configuration: 'api',
+      target       : 'react-native',
+      exclude      : [
+          'expo-firebase-analytics',
```

Comment:
do we need to exclude these everywhere (inc. here)? Or does it suffice to exclude them only in tools/android-tasks.js?

Reply:
The simplest solution is to remove unimodule.json from those packages, or add flag deprecated/disabled or sth similar.

In the current setup, we need to exclude in expoview/build.gradle only, and if we don't want to prepare prebuilts for  them also in android-tasks.

Reply:
Moreover, `expo-firebase-*` is to be removed, right? 🤔😔